archive/j/java/DepthFirstSearch.java

```diff
@@ -0,0 +1,92 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class DepthFirstSearch {
+
+    public static void main(String[] args) {
+        if (args.length != 3 || args[0].isEmpty() || args[1].isEmpty() || args[2].isEmpty()) {
+            handleError();
+            return;
+        }
+        int matrixSize = args[0].split(",").length;
+        if (matrixSize != Math.pow(args[1].split(",").length, 2)){
+            handleError();
+            return;
+        }
+        List<Integer> vertices = Stream.of(args[1].split(","))
+                .map(String::trim)
+                .map(Integer::parseInt)
+                .toList();
```

Comment:
Unfortunately, we are running Java 11 which does not seem to support the `toList` method. However, I tested your code and all of the current code with Java 23, and everything seemed to pass (with the exception of the error message having an invalid character). Please edit `archive/j/java/testinfo.yml` and change `11-slim` to `23-slim`.